### PR TITLE
[SENT 05/02/18] Digest

### PR DESCRIPTION
**Pull request by nvgrw**
review ples
[digest.pdf](https://github.com/icdocsoc/emails/files/1697128/digest.pdf)

### CODE REVIEW

File: xml/2018-02-05.xml [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+  <text>
+    Hello everyone,
+
+    Welcome to another week of the DoCSoc digest! It's even more-on-time this
+    week!
+
+    This week, we bring you a Microsoft Cognitive Services workshop!
+
+    **Also, you can sign up for our very own Valentine's Service, only until
+    Thursday 8th February. If here is someone special who you would like to send
```
Comment (paulbalaji) [inline xml/2018-02-05.xml]: "here" --> "there"

File: xml/2018-02-05.xml [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+  <text>
+    Hello everyone,
+
+    Welcome to another week of the DoCSoc digest! It's even more-on-time this
+    week!
+
+    This week, we bring you a Microsoft Cognitive Services workshop!
+
+    **Also, you can sign up for our very own Valentine's Service, only until
```
Comment (paulbalaji) [inline xml/2018-02-05.xml]: what's the ** for

File: xml/2018-02-05.xml [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+    Thursday 8th February. If here is someone special who you would like to send
+    a rose and message to,
+    [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSdeGAXfxl-DBHFd1xDdP3vMIRcP7oEWZWgJWSYt51-MdayGgw/viewform).
```
Comment (paulbalaji) [inline xml/2018-02-05.xml]: make it a link to the union shop item once we've got it up there

File: xml/2018-02-05.xml [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+  </event>
+
+  <heading>The Valentine's Bar Night</heading>
+  <event>
+    <facebook id="1607380125981757" />
+    <text>
+      It's time for the first Bar Night of 2018! Once again, we'll be meeting in
+      Metric at 7pm for an evening of free drinks and great conversation.
+
+      Drinks tokens will be given upon entry for FREE to any member of DoCSoc.
```
Comment (paulbalaji) [inline xml/2018-02-05.xml]: maybe mention the "if you've joined by X date"

File: xml/2018-02-05.xml [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+    role="DoCSoc Secretary"
+    facebook="nik.vangerow">
+    That's all for now!
+
+    Until next time,
+  </sign>
+
+  <text>
+    This week's biscuit brought to you by: Biscuit Voters!
+    [Submit Biscuit](https://docsoc.co.uk/suggest-biscuit).
```
Comment (paulbalaji) [inline xml/2018-02-05.xml]: pls can we have an actual biscuit instead of the pie

File: xml/2018-02-05.xml (hunk 2 above)
Comment (nvgrw) [reply xml/2018-02-05.xml]: It's markdown, emboldens text.

File: xml/2018-02-05.xml (hunk 4 above)
Comment (nvgrw) [reply xml/2018-02-05.xml]: It's on the fb event and 99% of the ppl reading it are long-time members so I don't think it's necessary. But happy to reconsider if nobody else agrees with me

File: xml/2018-02-05.xml (hunk 5 above)
Comment (nvgrw) [reply xml/2018-02-05.xml]: Feel free to suggest something 😛. It kinda works though because a pie chart is round lie a biscuit

**Pull request merged**